src/components/Presentation/PresentationHeader/Object/Image/index.jsx

```diff
@@ -0,0 +1,215 @@
+
+        switch (draggedVertexIndex) {
+          case 0:
+            newImageSpec = {
+              ...newImageSpec,
+              width: initialSpec.width - widthChange,
+              height: initialSpec.height - heightChange,
+              x: newPosition.x,
+              y: newPosition.y,
+            };
+            break;
+          case 2:
+            newImageSpec = {
+              ...newImageSpec,
+              width: initialSpec.width + widthChange,
+              height: initialSpec.height - heightChange,
+              y: newPosition.y,
+            };
+            break;
+          case 4:
+            newImageSpec = {
+              ...newImageSpec,
+              width: initialSpec.width + widthChange,
+              height: initialSpec.height + heightChange,
+            };
+            break;
+          case 6:
+            newImageSpec = {
+              ...newImageSpec,
+              width: initialSpec.width - widthChange,
+              height: initialSpec.height + heightChange,
+              x: newPosition.x,
+            };
+            break;
+          case 1:
+            newImageSpec = {
+              ...newImageSpec,
+              height: Math.max(10, initialSpec.height - heightChange),
+              y: newPosition.y,
+            };
+            break;
+          case 5:
+            newImageSpec = {
+              ...newImageSpec,
+              height: Math.max(10, initialSpec.height + heightChange),
+            };
+            break;
+          case 3:
+            newImageSpec = {
+              ...newImageSpec,
+              width: Math.max(10, initialSpec.width + widthChange),
+            };
+            break;
+          case 7:
+            newImageSpec = {
+              ...newImageSpec,
+              width: Math.max(10, initialSpec.width - widthChange),
+              x: newPosition.x,
+            };
+            break;
+          default:
+            break;
+        }
+
+        setImageSpec(newImageSpec);
+      };
+
+      document.addEventListener("mousemove", moveHandler);
+      document.addEventListener(
+        "mouseup",
+        () => {
+          document.removeEventListener("mousemove", moveHandler);
+        },
+        { once: true },
+      );
+    },
+    [imageSpec],
+  );
+
+  const onImageDrag = (e) => {
+    e.stopPropagation();
+    e.preventDefault();
+
+    const initialPosition = {
+      x: e.clientX - imageSpec.x,
+      y: e.clientY - imageSpec.y,
+    };
+
+    const moveHandler = (moveEvent) => {
+      setImageSpec((prevSpec) => ({
+        ...prevSpec,
+        x: moveEvent.clientX - initialPosition.x,
+        y: moveEvent.clientY - initialPosition.y,
+      }));
+    };
+
+    const upHandler = () => {
+      document.removeEventListener("mousemove", moveHandler);
+      document.removeEventListener("mouseup", upHandler);
+    };
+
+    document.addEventListener("mousemove", moveHandler);
+    document.addEventListener("mouseup", upHandler);
+  };
```

Comment:
```suggestion
    function handleMouseMove(moveEvent) {
      setImageSpec((prevSpec) => ({
        ...prevSpec,
        x: moveEvent.clientX - initialPosition.x,
        y: moveEvent.clientY - initialPosition.y,
      }));
    };

    function handleMouseUp() {
      document.removeEventListener("mousemove", moveHandler);
      document.removeEventListener("mouseup", upHandler);
    };

    document.addEventListener("mousemove", moveHandler);
    document.addEventListener("mouseup", upHandler);
  };
```
이벤트 핸들러 선언문으로 바껴야 할 것 같습니다 !